.env.example

```
@@ -0,0 +1,23 @@
+NODE_ENV=development # or 'production'
+PORT=3000
+LISTEN_ADDRESS=0.0.0.0
+
+SESSION_SECRET=not-secure-change-me # an UUID or `openssl rand -hex 32` or something similar
```

dotenv-linter (4.0.0)

[warning] [ValueWithoutQuotes] This value needs to be surrounded in quotes

(ValueWithoutQuotes)